src/main/java/com/raoulvdberge/refinedstorage/api/autocrafting/ICraftingPatternContainer.java

```diff
@@ -53,12 +68,29 @@
     BlockPos getPosition();
 
     /**
-     * @return true if this container is blocked, false otherwise
+     * Containers may be daisy-chained together.  If this container points to
+     * another one, gets the last container in the chain.  If containers are
```

Comment:
last container? Don't you mean first container? The root one (the one at front) is the first one.